### PR TITLE
Base the PNI template off of the base template

### LINKED ISSUE #3675: Implement Fo'Mo Nav onto Privacy Not Included
Change Privacy Not Included's Nav (just the navigation bar, not the menu style) to match Fo'Mo.

---

**🗒  Design -> Dev Handoff Checklist**
(Feel free to remove items that are not applicable.)

- [x] Design has been finalized
- [x] Link to original design ticket: #3614 
- [x] Link to Redpen: https://redpen.io/rhd7b2c19478a1fa67
- [x] Design changes been merged into the master Sketch file
- [x] Desktop and mobile mockups are included

Home@2x

### PR DESCRIPTION
**Pull request by Pomax**
url:
- https://foundation-mofostaging-pr-3702.herokuapp.com/en/privacynotincluded/
- https://foundation-mofostaging-pr-3702.herokuapp.com/en

looks like I still missed a CSS class somewhere, because it's not doing the show/reveal for the newsletter. Instead it's very much just "always visible"

Closes #3675

### CODE REVIEW

**Comment by mmmavis** [comment]
@Pomax we have to use `nav-newsletter.js` for the nav to work on both mobile and desktop. See example: https://github.com/mozilla/foundation.mozilla.org/blob/master/source/js/main.js#L190

File: source/js/nav-newsletter.js [hunk 1]
```diff
@@ -53,8 +56,8 @@ class NavNewsletter {
 
   // For desktop+ version:
   // transition newsletter section to its expanded state
-  expandDesktopNewsletter(event) {
-    elements.container.style.top = `${elements.primaryNav.offsetHeight}px`;
+  expandDesktopNewsletter(eventevent) {
```
Comment (Pomax) [inline source/js/nav-newsletter.js]: what...?

**Comment by Pomax** [comment]
@mavis this should be ready for code review, everything should effectively inherit from the foundation nav now, so code review and design review should be possible in parallel.

File: network-api/networkapi/buyersguide/templates/bg_base.html [hunk 2]
```diff
@@ -1,71 +1,100 @@
...
+
+{% block social_metadata %}
+  <meta property='og:type' content='website'>
+  <meta property='og:locale' content='en-US'>
+  <meta property="og:title" content="*Privacy Not Included: A Buyer’s Guide for Connected Products">
+{% endblock %}
+
+
+{% block wagtail_metadata %}
+  <meta property="og:description" content="89 connected products rated on their safety and security">
```
Comment (mmmavis) [inline network-api/networkapi/buyersguide/templates/bg_base.html]: Is it still 89 now that we only show recently reviewed products?

File: network-api/networkapi/buyersguide/templates/bg_base.html [hunk 3]
```diff
@@ -1,71 +1,100 @@
...
+
+{% block icons %}
+  <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="/_images/favicons/apple-touch-icon-180x180@2x.png">
+  <link rel="icon" type="image/png" sizes="196x196" href="/_images/favicons/favicon-196x196@2x.png">
+  <link rel="shortcut icon" href="/_images/favicons/favicon.ico">
+{% endblock %}
+
+
+{% block additional_head_elements %}
+  <!-- do we still need this? -->
```
Comment (mmmavis) [inline network-api/networkapi/buyersguide/templates/bg_base.html]: I'm not sure what this was originally for.

File: source/js/nav-newsletter.js [hunk 4]
```diff
@@ -105,6 +111,7 @@ class NavNewsletter {
       // find this DOM node, and report on the result (binding it if found for later use)
       let element = document.querySelector(value);
       if (element) elements[key] = element;
+      console.log(key, element);
```
Comment (mmmavis) [inline source/js/nav-newsletter.js]: 😬

File: source/js/nav-newsletter.js [hunk 5]
```diff
@@ -14,6 +14,8 @@ const elements = {
   buttonDismiss: `#nav-newsletter-form-wrapper .form-dismiss`
 };
 
+console.log(elements);
```
Comment (mmmavis) [inline source/js/nav-newsletter.js]: 😮

File: source/js/buyers-guide/components/primary-nav/primary-nav.scss [hunk 6]
```diff
@@ -144,3 +147,7 @@
     }
   }
 }
+
+#nav-newsletter-form-wrapper.expanded {
+  z-index: 11; /* TODO: is there a var that can be used for this? */
```
Comment (mmmavis) [inline source/js/buyers-guide/components/primary-nav/primary-nav.scss]: We use many arbitrary z-index values for Buyer's Guide so I don't know at this point replacing `11` with any [Bootstrap z-index variables](https://getbootstrap.com/docs/4.3/layout/overview/#z-index) can help much?

**Comment by beccaklam** [comment]
> Left some comments.
> 
> I think we can remove `border-top: 1px solid $gray-20;` from `.category-nav { ... }` as currently the divider in between primary nav and category nav looks thicker than what we have on main site.
> 
> nav
> 
> Nav on mobile is taller compared to main site. Not sure if it's intentional? Let's see what @beccaklam says.
> 
> mobile-nav

Oh that is not intentional ...

File: network-api/networkapi/buyersguide/templates/bg_base.html [hunk 7]
```diff
@@ -1,71 +1,100 @@
...
-              <a class="{% bg_active_nav request.get_full_path home_url %}" href="{{ home_url }}">Home</a>
-              {% category_nav request.get_full_path category categories %}
-              {% url 'about-view' as about_url %}
-              <a class="{% bg_active_nav request.get_full_path about_url %}" href="{{ about_url }}">About the Guide</a>
-            </div>
+
+{% block page_title %}{% if page_title %}{{ page_title }}{% else %}Mozilla - *privacy not included{% endif %}{% endblock %}
+
+
+{% block ga_identifier %}
```
[the remark this answers is not in the document]
Comment (Pomax) [reply network-api/networkapi/buyersguide/templates/bg_base.html]: we don't, good catch

File: network-api/networkapi/buyersguide/templates/bg_base.html (hunk 2 above)
Comment (Pomax) [reply network-api/networkapi/buyersguide/templates/bg_base.html]: this should now be 75

File: network-api/networkapi/buyersguide/templates/bg_base.html [hunk 8]
```diff
@@ -1,71 +1,100 @@
...
+  <meta property="og:image:height" content="628">
+  <meta property="og:url" content="{{request.scheme}}://{{request.get_host}}{{request.path}}">
+{% endblock %}
+
+
+{% block stylesheets %}
+  <link rel="stylesheet" href="/_css/buyers-guide.compiled.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Nunito+Sans:400,300,700,300i">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Zilla+Slab:300,400,600,700,300i">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,700">
```
[the remark this answers is not in the document]
Comment (Pomax) [reply network-api/networkapi/buyersguide/templates/bg_base.html]: I'm okay leaving it as three distinct fonts, though. Having them separate does not significantly impact page load.

File: network-api/networkapi/buyersguide/templates/bg_base.html (hunk 3 above)
Comment (Pomax) [reply network-api/networkapi/buyersguide/templates/bg_base.html]: I'll leave it in for now and we can clean it up once we figure out what it was for...

File: network-api/networkapi/buyersguide/templates/primary_nav.html [hunk 9]
```diff
@@ -1,46 +1,40 @@
...
-            </div>
-          </div>
-        </div>
-      </div>
+{% block narrow_screen_menu %}
+  <div class="narrow-screen-menu-background">
+    <div class="narrow-screen-menu-container">
 
-      <div class="container">
+      {# TODO: DO WE STILL NEED THIS INCLUDE NOW? #}
```
[the remark this answers is not in the document]
Comment (Pomax) [reply network-api/networkapi/buyersguide/templates/primary_nav.html]: :100:

File: network-api/networkapi/settings.py [hunk 10]
```diff
@@ -195,8 +195,8 @@
 
     # wagtail-specific app prefixed so that it can be localised
     'networkapi.wagtailpages',
-    'networkapi.buyersguide',
     'networkapi.mozfest',
+    'networkapi.buyersguide',
```
[the remark this answers is not in the document]
Comment (Pomax) [reply network-api/networkapi/settings.py]: it very much does: with this ordering, buyersguide can already load mozfest objects, even though things after it have not been loaded in yet.

File: source/js/buyers-guide/components/primary-nav/primary-nav.scss (hunk 6 above)
Comment (Pomax) [reply source/js/buyers-guide/components/primary-nav/primary-nav.scss]: I guess I'll leave it as `11` with a comment to explain why

**Comment by Pomax** [comment]
@mmmavis @beccaklam I've edited the scss and html template to fix the container border in general, and spacing for mobile, as well as all the other issues flagged. This PR is read for re-review

File: network-api/networkapi/buyersguide/templates/primary_nav.html [hunk 11]
```diff
@@ -1,46 +1,37 @@
...
-</div>
+{% endblock %}
+
+{% block burger_and_logo %}
+  {# we do not want the burger and logo from the original nav here #}
+{% endblock %}
+
+{% block wide_screen_menu %}
+  <div class="d-flex align-items-center justify-content-between">
+    <div class="d-flex align-items-center logo-section">
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/primary_nav.html]:
```suggestion
    <div class="d-flex align-items-center py-3 py-s-0 logo-section">
```

File: network-api/networkapi/buyersguide/templates/primary_nav.html [hunk 12]
```diff
@@ -1,46 +1,37 @@
...
-</div>
+{% endblock %}
+
+{% block burger_and_logo %}
+  {# we do not want the burger and logo from the original nav here #}
+{% endblock %}
+
+{% block wide_screen_menu %}
+  <div class="d-flex align-items-center justify-content-between">
+    <div class="d-flex align-items-center py-3 py-s-0 logo-section">
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/primary_nav.html]:
```suggestion
    <div class="d-flex align-items-center py-0 py-md-3 logo-section">
```

**Comment by kristinashu** [comment]
Agree, 2px is not a blocker!

**Pull request merged**